src/assessments/pointer-motion/test-steps/motion-operation.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NewTabLink } from '../../../common/components/new-tab-link';
+import { link } from '../../../content/link';
+import { ManualTestRecordYourResults } from '../../common/manual-test-record-your-results';
+import { Requirement } from '../../types/requirement';
+import { PointerMotionTestStep } from './test-steps';
+
+const description: JSX.Element = <span>Users must be able to cancel functions that can be operated using a single pointer.</span>;
+
+const howToTest: JSX.Element = (
+    <div>
+        <ol>
+            <li>
+                Examine the target page to identify any functions that can be operated by device motion (such as shaking) or user motion
+                (such as walking).
+            </li>
+            <li>
+                Verify that both of the following are true:
+                <li>The function is also operable through user interface components (such as a toggle button). </li>
```

Comment:
Looks like we don't have e2e test to cover this content just yet. I notice we have a child `<li>` with a `<li>` parent here and I recall this fail the `listitem` rule, so I run the self validator to make sure and got this:
![image](https://user-images.githubusercontent.com/2837582/61730903-370a4980-ad2f-11e9-902f-cec07c79fe09.png)

I think there are other places where we have `<li>` -> `<li>` structures that'll need to be fix too. I'll go ahead and make a smaller comment on the ones that I find to make it easier to review them too.